alexey.zhuravlov/task1/MyHashMap/src/MyHashMap.java

```diff
@@ -0,0 +1,53 @@
+import java.util.Objects;
+
+public class MyHashMap
+{
```

Comment:
Please change your code style.
Correct example is public class MyHashMap {
Don't break lines with symbol {
Press ctrl+alt+shift+l to fix code style